packages/fuels-abigen-macro/tests/harness.rs

```diff
@@ -2034,3 +2039,44 @@ async fn can_use_try_into_to_construct_enum_from_bytes() -> Result<(), Error> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn test_init_storage_automatically() -> Result<(), Error> {
+    abigen!(
+        MyContract,
+        "packages/fuels-abigen-macro/tests/test_projects/contract_storage_test/out/debug/contract_storage_test-abi.json"
+    );
+
+    let wallet = launch_provider_and_get_wallet().await;
+
+    // ANCHOR: automatic_storage
+    let contract_id = Contract::deploy_with_parameters(
```

Comment:
Will the `deploy()` function not allow passing in a path to the storage slots JSON? The reason I ask is because `forc new` generates a test harness that uses `deploy()` and we're probably going to have to change that to `deploy_with_parameters()` to minimize the amount of work users have to do. This means that most tests out there will eventually be using `deploy_with_parameters()` instead of `deploy()`.

Reply:
Now we have it in both functions!